notify/impl_test.go

```diff
@@ -218,12 +220,28 @@ func readBody(t *testing.T, r *http.Request) string {
 }
 
 func TestOpsGenie(t *testing.T) {
+	templateDir, err := ioutil.TempDir("", "opsgenie-")
```

Comment:
Why are you doing all this setup? You can test this like the others.

Reply:
Ah, you mean to create a file in testdata? Didn't occur to me :see_no_evil: Will do!

Reply:
You should't need to create any files. The use case here is that the apikey is coming from an alert label, and any random template expression is sufficient to verify that templating is being applied.